lib/view/tabs/community/community_tab.dart

```diff
@@ -17,13 +16,7 @@ class CommunityTab extends StatefulWidget {
 }
 
 class _CommunityTabState extends State<CommunityTab> {
-  List<no.Notification> notifications =
-  []; // TODO: get notifications from controller
-
-  void _tapNotifs() {
-    Navigator.push(context,
-        MaterialPageRoute(builder: (context) => const NotificationsView()));
-  }
+  final String _testUserID = "1"; // TODO READ USER ID OF VIEWER FROM SOMEWHERE?
```

Comment:
not sure where to get the user ID of the current user?

Reply:
from the userManager